app/models/organization.rb

```diff
@@ -80,6 +80,14 @@ def login_settings
     @login_settings ||= Mumukit::Auth::LoginSettings.new(login_methods)
   end
 
+  def absolute_link
+    "http://#{relative_link}"
+  end
+
+  def relative_link
+    "#{name}.mumuki.io"
+  end
```

Comment:
Also, this means that we are assuming atheneum is deployed on `mumuki.io`, but that is not necessary true. 

I know we have a lot of code that assumes that, but I think we should avoid when possible adding new code that assumes it from now on, shouldn't we?

We could parameterize such domain in te environments configuration files.